packages/valory/skills/hello_world_abci/rounds.py

```diff
@@ -152,14 +162,34 @@ class ResetAndPauseRound(CollectSameUntilThresholdRound, HelloWorldABCIAbstractR
     def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
         """Process the end of the block."""
         if self.threshold_reached:
-            return self.synchronized_data.create(), Event.DONE
+            # TODO: Question: Seems like we can just not re-create the synchronized data here, to keep the print_count alive. But is that the right approach?
```

Comment:
Tip: By design, the synchronized data clears all keys unless they are explicitly marked as "cross-period persistent keys" in the AbciApp.

Reply:
Thanks! That works 🎉